internal/api/auth.go

```diff
@@ -96,6 +108,48 @@ func Signup(s storage.Store) http.HandlerFunc {
 	}
 }
 
+// Confirm ...
+func Confirm(s storage.Store) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		email := mux.Vars(r)["email"]
+		code := mux.Vars(r)["code"]
+		usr, err := s.Users().FindByEmail(email)
+		if err != nil {
+			errs := []string{"Email not found!", "Raw error: " + err.Error()}
+			message := "Email couldn't confirm!"
+			RespondWithErrors(w, http.StatusBadRequest, message, errs)
+			return
+		} else if !usr.EmailVerifiedAt.IsZero() {
+			errs := []string{"Email is already verified!"}
+			message := "Email couldn't confirm!"
+			RespondWithErrors(w, http.StatusBadRequest, message, errs)
+			return
+		} else if code != usr.ConfirmationCode {
+			errs := []string{"Confirmation code is wrong!"}
+			message := "Email couldn't confirm!"
+			RespondWithErrors(w, http.StatusBadRequest, message, errs)
+			return
+		}
+
+		updatedUser := model.ToUserDTO(usr)
+		updatedUser.EmailVerifiedAt = time.Now()
+
+		_, err = app.UpdateUser(s, usr, updatedUser, false)
+		if err != nil {
+			errs := []string{"User can't updated!", "Raw error: " + err.Error()}
+			message := "Email couldn't confirm!"
```

Comment:
Same here, messages should be read from localization files

Reply:
As I told before, localization is still in progress.